src/web/routes/application/common/test/apply-express-validation.js

```diff
@@ -26,6 +26,9 @@ const applyExpressValidation = async (req, middleware) => {
     locals: {}
   }
 
+  // Ensure translation function is applied to request object
+  req.t = str => str
+
```

Comment:
This express validation test helper is specific to our app setup. Adding translation function to request as it is required for the tests to run correctly - this stops it being omitted by accident.